test_performance.py

```diff
@@ -15,17 +20,52 @@
 
 
 # Print iterations progress
-def printProgressBar (iteration, total, decimals = 1, fill = '█', printEnd = "\r"):
+def printProgressBar(iteration, total, decimals=1, fill='█', printEnd="\r"):
     percent = ("{0:." + str(decimals) + "f}").format(100 * (iteration / float(total)))
     filledLength = int(100 * iteration // total)
     bar = fill * filledLength + '-' * (100 - filledLength)
-    print(f'\r |{bar}| {percent}% ', end = printEnd)
-    # Print New Line on Complete
+    print(f'\r |{bar}| {percent}% ', end=printEnd)
     if iteration == total:
         print()
 
 
+def predict_sota(text):
+    payload = {
+        "language": "en",
+        "sentences": False,
+        "text": text,
+        "version": "latest"
+    }
+    headers = {
+        "Authorization": f"Bearer {WINSTON_API_KEY}",
+        "Content-Type": "application/json"
+    }
+
+    response = requests.request("POST", "https://api.gowinston.ai/functions/v1/predict", json=payload, headers=headers)
+    case = json.loads(response.text)["score"]
+    if case <= 40:
+        return 1
+    elif case >= 60:
+        return -1
+    else:
+        return 0
+
+
+def calculate_metrics(predictions, source):
+    true = predictions[source_mapping[source]]
+    unsure = predictions[0]
+    false = predictions[-source_mapping[source]]
+    print(prediction, " true: ", str(round(true / count * 100, 2)), "%")
+    print(prediction, " unsure: ", str(round(unsure / count * 100, 2)), "%")
+    print(prediction, " false: ", str(round(false / count * 100, 2)), "%\n")
```

Comment:
is `count` the variable that we happen to define in line 136 calling this function? this seems a bit adventurous even for this script😂 maybe let's at least pass it as an argument